ci.sh

```diff
@@ -176,6 +176,21 @@ case "$cmd" in
       'run x-release amd64' \
       'run a-release arm64' | DUP=1 cache_log "Release CI run" $RUN_ID
     ;;
+  "release-test")
+    prep_vars
+    # Spin up ec2 instance and run the release test flow.
+    run() {
+      JOB_ID=$1 INSTANCE_POSTFIX=$1 ARCH=$2 exec denoise "bootstrap_ec2 './bootstrap.sh $3'"
+    }
+    export -f run
+    # We need to run fast CI and release test flow on both x86 and arm64.
+    # The release flow doesn't run tests, so we run them too.
+    parallel --termseq 'TERM,10000' --tagstring '{= $_=~s/run (\w+).*/$1/; =}' --line-buffered --halt now,fail=1 ::: \
+      'run x-fast amd64 ci-fast' \
+      'run a-fast arm64 ci-fast' \
```

Comment:
I had thought that you might keep ci-release-test on the PR so i also had testing here